modules/multi-architecture-verifying-cluster-compatibility.adoc

```diff
@@ -18,6 +23,18 @@ Before you can start adding compute nodes of different architectures to your clu
 
 * You installed the OpenShift CLI (`oc`)
 
+ifdef::ibm-power[]
+[NOTE]
+====
+When using multiple architectures, hosts for the {product-title} nodes must share the same storage layer. If they do not have the same storage layer, use a storage provider such as `nfs-provisioner`.
+====
+
+[NOTE]
+====
+One should limit the number of network hops between the compute and control plane as much as possible.
```

Comment:
The use of "one" seems strange to me. Is this referring to the user? I would replace with "you" if so.

Reply:
I dont see anything against it in the style guides, but it seems it would be clearer if you used "you".

Reply:
Agree. Updated the PR.